changelog/fragments/1714505172-Send-fleet-server-component-output-under-bootstrap-key.yaml

```diff
@@ -0,0 +1,39 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: Send fleet-server component output under bootstrap key
```

Comment:
Make this less technical, be more user facing. Give a quick benefit description.

`Fleet Server component now uses policy output configuration to communicate with elasticsearch`